### ISSUE TITLE
[ProjectManagement] Sprint PLanning Template

### ISSUE THREAD

**Issue by aolite** [AmplifyForPeople/android-project#3]
Despite the sprint planning is well described. We need an image regarding the Model Kanban. And specifically, how the Product Backlog is finally implemented in the Kanban with the corresponding task estimates.

**Comment by ffriz22** [comment]
Hi @aolite ! You mean this [Sprint 1 -  wiki](https://github.com/AmplifyForPeople/amplify-service/wiki/Sprint-Planning-(31-10-2018)) And about the estimates, are they not described in the issue?

**Comment by aolite** [comment]
No, I refer to the product backlog (where all general issues are located). The product backlog should be estimated to help us to specify the task in the sprint backlog (once the product backlog task is selected to be developed). 

So, you need to do the following tasks: 

- [ ] Introduce the user stories into the product backlog.

- [ ] Estimate the tasks of the product backlog.

After that, you will be ready to start with the Amplify App development :)
